content/ADR-188-favorite-places.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: adr
+adr: 188
+title: Favorite Places
+date: 2023-02-07
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - davidejensen
+---
+
+## Abstract
+
+This document describes an approach on how to integrate the new Favorite Places functionality in Decentraland.
```

Comment:
Please expand the abstract to summarize the content of the document (or ask chatgpt to do it for you)